Sources/StreamChatUI/ChatMessageList/Reactions/ChatMessageReactionAuthorsVC/ChatMessageReactionAuthorViewCell.swift

```diff
@@ -0,0 +1,146 @@
+    open lazy var reactionBubbleView: ChatReactionBubbleBaseView = components
+        .messageReactionsBubbleView.init()
+        .withoutAutoresizingMaskConstraints
+
+    /// The reaction view inside the reaction bubble.
+    public lazy var reactionItemView: ChatMessageReactionItemView = components
+        .messageReactionItemView.init()
+        .withoutAutoresizingMaskConstraints
+
+    /// The constraint that if active renders the reaction in the leading side of the avatar view.
+    public var reactionLeadingConstraint: NSLayoutConstraint?
+
+    /// The constraint that if active renders the reaction in the trailing side of the avatar view.
+    public var reactionTrailingConstraint: NSLayoutConstraint?
+
+    /// The size of the avatar view that belongs to the author of the reaction.
+    open var authorAvatarSize: CGSize { .init(width: 64, height: 64) }
+
+    override open func setUpAppearance() {
+        super.setUpAppearance()
+
+        authorNameLabel.font = appearance.fonts.footnoteBold
+        authorNameLabel.textAlignment = .center
+        authorNameLabel.numberOfLines = 2
+        authorNameLabel.adjustsFontSizeToFitWidth = true
+    }
+
+    override open func setUpLayout() {
+        super.setUpLayout()
+
+        containerStack.axis = .vertical
+        containerStack.alignment = .top
+        containerStack.spacing = 8
+        containerStack.distribution = .natural
+
+        contentView.addSubview(containerStack)
+        NSLayoutConstraint.activate([
+            containerStack.leadingAnchor.pin(equalTo: contentView.leadingAnchor),
+            containerStack.trailingAnchor.pin(equalTo: contentView.trailingAnchor),
+            containerStack.topAnchor.pin(equalTo: contentView.topAnchor)
+        ])
+
+        containerStack.addArrangedSubview(authorAvatarView)
+        containerStack.addArrangedSubview(authorNameLabel)
+        authorAvatarView.addSubview(reactionBubbleView)
+
+        reactionBubbleView.addSubview(reactionItemView)
+        reactionItemView.pin(to: reactionBubbleView.layoutMarginsGuide)
+
+        NSLayoutConstraint.activate([
+            authorAvatarView.widthAnchor.pin(equalToConstant: authorAvatarSize.width),
+            authorAvatarView.heightAnchor.pin(equalToConstant: authorAvatarSize.height),
+            authorNameLabel.widthAnchor.pin(equalTo: authorAvatarView.widthAnchor),
+            reactionBubbleView.bottomAnchor.pin(equalTo: authorAvatarView.bottomAnchor)
+        ])
+
+        reactionTrailingConstraint = reactionBubbleView.rightAnchor.pin(equalTo: authorAvatarView.centerXAnchor)
+        reactionLeadingConstraint = reactionBubbleView.leftAnchor.pin(equalTo: authorAvatarView.centerXAnchor)
+
+        reactionTrailingConstraint?.isActive = false
+        reactionLeadingConstraint?.isActive = false
+    }
+
+    override open func updateContent() {
+        super.updateContent()
+
+        guard let content = self.content else { return }
```

Comment:
I don't think it's a good pattern, tbh. If `nil` is assigned I would expect the view to display nothing but not showing the previous content. It's especially useful when the cell is being prepared for reuse